Alignment/CommonAlignment/python/tools/trackselectionRefitting.py

```diff
@@ -198,8 +213,10 @@ def getSequence(process, collection,
                 "minimumHits": 10,
                 })
     else:
-        print "Unknown input track collection:", collection
-        sys.exit(1)
+        raise ValueError("Unknown input track collection: {}".format(collection))
+
+    if cosmicTrackSplitting and not isCosmics:
+        raise ValueError("Can only do cosmic track splitting for cosmics.")
```

Comment:
I don't know, this is what I usually use ValueErrors for... the documentation says "argument that has the right type but an inappropriate value," which sounds reasonable (even if it's the combination of two arguments that has the wrong value).